src/Samples/CSharp/ConvertToAutoProperty/Impl/PropertyRewriter.cs

```diff
@@ -57,10 +57,10 @@ public override SyntaxNode VisitPropertyDeclaration(PropertyDeclarationSyntax pr
 
         public override SyntaxNode VisitFieldDeclaration(FieldDeclarationSyntax field)
         {
-            // Retrieve the symbol for the field
+            // Retrieve the symbol for the field's variable
             if (field.Declaration.Variables.Count == 1)
             {
-                if (object.Equals(semanticModel.GetDeclaredSymbol(field), backingField))
+                if (object.Equals(semanticModel.GetDeclaredSymbol(field.Declaration.Variables.FirstOrDefault()), backingField))
```

Comment:
Nitpick: I'd call .Single() since you've already established there's exactly one. Sometimes FirstOrDefault is used to mean "I haven't thought about which one I should pick" and so is sometimes a code smell.